bigchaindb/backend/schema.py

```diff
@@ -21,9 +21,11 @@
 logger = logging.getLogger(__name__)
 
 TABLES = ('bigchain', 'backlog', 'votes', 'assets')
-VALID_LANGUAGES = ('danish' 'dutch' 'english' 'finnish' 'french' 'german'
-                   'hungarian' 'italian' 'norwegian' 'portuguese' 'romanian'
-                   'russian' 'spanish' 'swedish' 'turkish')
+VALID_LANGUAGES = ('danish', 'dutch', 'english', 'finnish', 'french', 'german',
+                   'hungarian', 'italian', 'norwegian', 'portuguese', 'romanian',
+                   'russian', 'spanish', 'swedish', 'turkish',
+                   'da', 'nl', 'en', 'fi', 'fr', 'de', 'hu', 'it', 'nb', 'pt',
+                   'ro', 'ru', 'es', 'sv', 'tr')
```

Comment:
There are still some languages missing here: `ara`, `prs`, `pes`, `urd`, `zhs`, and `zht` (from https://docs.mongodb.com/manual/reference/text-search-languages/ ).

Reply:
`none` is also a valid value for language.

Reply:
I wonder if the long-form versions of those other languages also works, e.g. `traditional chinese`

Reply:
Okay we decided to allow only those languages supported by MongoDB community edition (so not `ara` etc.).

`'none'` still needs to be added to the above.

Reply:
Sure, I will add `'none'` to the list and update asset docs.